Articles/Blog/2022-08-brightspot-ci.md

```diff
@@ -0,0 +1,93 @@
+# Bright Spots: Team Experiences Implementing Continuous Integration
+
+**Hero Image:**
+
+- <img src='../../images/Blog_Brightspot.png' />
+
+#### Contributed by [David M. Rogers](https://github.com/frobnitzem)
+
+#### Publication date: August 12, 2022
+
+*This is the first in an occasional series of Bright Spots blog articles, highlighting success stories of Exascale Computing Project software teams implementing and benefitting from software development best practices.  If you'd like to contribute to a future Bright Spot article, please contact the [BSSw Editorial Board](mailto:bssw-editorial@lists.mcs.anl.gov?subject=Bright Spots).*
+
+[Continuous Integration](https://bssw.io/items/what-is-continuous-integration-testing) is an enabling technology that allows code development teams to insert testing directly into their development workflow[2] .  Although there are many advantages to CI, including more stable working code bases, faster test feedback with less distraction, and greater support for community contributions, there are also some pitfalls[3] , like initial adoption and support burden, disruptive changes to team processes, and a potentially disorienting array of competing, overlapping, new technologies.
```

Comment:
@frobnitzem, we need to do something with the [2] and [3].  We discourage the use of references in blog articles, but we can do that if they're really needed.  But think about whether you really need them here.